sale_blanket_order/views/sale_blanket_order_views.xml

```diff
@@ -284,7 +271,7 @@
                                     <t groups="base.group_multi_company">
                                         <field
                                             name="analytic_account_id"
-                                            attrs="{'readonly': [('sale_count','!=',0),('state','!=','draft')]}"
+                                            readonly="sale_count != 0 or state != 'draft'"
```

Comment:
Shouldn't the `or` be and `and` as no or-operator (`'|'`) is present in de `attrs` domain?